ide/api.lsp/nbproject/project.properties

```diff
@@ -15,7 +15,8 @@
 # specific language governing permissions and limitations
 # under the License.
 is.autoload=true
-javac.source=1.8
+javac.source=11
+javac.target=11
```

Comment:
`javac.release` is now possible (https://github.com/apache/netbeans/pull/7247) since both NB 22 and the build support it.

(the only exception are modules which use nb-javac, but a PR is pending for it)

Reply:
Fix merged into 54779f9df79a